packages/connector/src/signers/index.ts

```diff
@@ -90,6 +91,10 @@ export class SignersController implements ReactiveController {
       this.addSigner("Nostr", "Nostr", NOSTR_SVG, nip07Signer);
     }
 
+    ccc.UtxoGlobal.getUtxoGlobalSigners(client)?.forEach(({ signer, name }) => {
```

Comment:
Since it returns an array, it should always be not null. (But can be empty)

Reply:
fixed